pkg/gui/services/custom_commands/session_state_loader.go

```diff
@@ -20,8 +21,16 @@ func NewSessionStateLoader(c *helpers.HelperCommon, refsHelper *helpers.RefsHelp
 	}
 }
 
-type Commit struct {
-	Hash          string
+// We create shims for all the model classes in order to get a more stable API
```

Comment:
Great stuff. We should also update docs/Custom_Command_Keybindings.md which currently says 
> To see what fields are available on e.g. the `SelectedFile`, see [here](https://github.com/jesseduffield/lazygit/blob/master/pkg/commands/models/file.go)

We should update this to just point to this file now

Reply:
This is a tricky one. In some cases the original model struct has documentation for its fields (e.g. `models.Branch`), which I didn't copy over to the shims. So pointing users to the shims gives them less information.

Or should I copy the comments over? Seemed so redundant.

Reply:
hmm, interesting. I say point the users to this file but don't copy the comments. Most fields are self-explanatory and the ones that aren't are likely not the ones the user needs

Reply:
OK, works for me. To make this easier for users, I decided to move the struct definitions to a separate file (7ab7fae99a66), and to remove the "Shim" from the names again (416106a59da4). Documentation is adapted here: 91132a7321d8.

I noticed that the documentation already says that we don't guarantee backwards compatibility. Do we now, and should remove this sentence? Also, it says we are planning to expose a tighter API in the future; do we want to change anything about that sentence? Feel free to push a fixup if you want to tweak that documentation further.

Reply:
I suppose we now do support backwards compatibility so indeed we can lose that line. We can also remove the part about the tighter API: I have no plans for changing things anytime soon.

Reply:
OK, removed in 283cf3a3b64e.

Going to squash now, and then merge (will take a moment as the fixups conflict).